File: src/test/resources/csharp/benchmark/Domain/Indirect/AllowedFrom/AccessObjectReferenceIndirect_AsReturnValue_MethodDerivedViaArgumentType.cs
using Domain.Indirect;

namespace Domain.Indirect.AllowedFrom
{

	public class AccessObjectReferenceIndirect_AsReturnValue_MethodDerivedViaArgumentType

	{
	public string initializeProfileInformation() {
		String profile = "profiled";
		BaseIndirect baseClass = new BaseIndirect();
		Object o = baseClass.getMethodCorrectlyByDerivedArgumentType("tsja", profile, 3);
		return o;	}

	}
}

File: testresources/java/accuracy/src/domain/indirect/allowedfrom/AccessObjectReferenceIndirect_AsReturnValue_MethodDerivedViaArgumentType.java
package domain.indirect.allowedfrom;

import domain.indirect.BaseIndirect;

public class AccessObjectReferenceIndirect_AsReturnValue_MethodDerivedViaArgumentType {
	
	public String initializeProfileInformation(){
		String profile = "profiled";
		BaseIndirect baseClass = new BaseIndirect();
		Object o = baseClass.getMethodCorrectlyByDerivedArgumentType("tsja", profile, 3);
		return (String) o;
	}
}